### PR TITLE
fix: check cloud run jobs status in CFT test

### PR DESCRIPTION
**Pull request by CienetStingLin**
Added a CFT test case to check the execution status of Cloud Run jobs to avoid race conditions, where it is attempting to destroy an active Cloud Run job

### CODE REVIEW

File: infra/test/integration/simple_example/simple_example_test.go [hunk 1]
```diff
@@ -58,6 +58,39 @@ func TestSimpleExample(t *testing.T) {
...
+			var jobResults []bool
+			for _, cloudRunJob := range cloudRunJobs {
+				jobName := cloudRunJob.Get("metadata.name").String()
+				jobCompletionTime := cloudRunJob.Get("status.latestCreatedExecution.completionTimestamp").String()
+				jobExecutionCount := cloudRunJob.Get("status.executionCount").Int()
+				fmt.Printf("Job %s completion time: %s, executionCount: %d\n", jobName, jobCompletionTime, jobExecutionCount)
+				if jobCompletionTime == "" || jobExecutionCount < 1 {
+					fmt.Printf("Job %s is not completed\n", jobName)
+					jobResults = append(jobResults, false)
+				} else {
```
Comment (telpirion) [inline infra/test/integration/simple_example/simple_example_test.go:74]: nit: I would rewrite this without the `else` statement.

```
if jobCompletionTime == "" || jobExecutionCount < 1 {
	fmt.Printf("Job %s is not completed\n", jobName)
	jobResults = append(jobResults, false)
        continue
}

fmt.Printf("Job %s is completed\n", jobName)
jobResults = append(jobResults, true)
```

File: infra/test/integration/simple_example/simple_example_test.go [hunk 2]
```diff
@@ -58,6 +58,39 @@ func TestSimpleExample(t *testing.T) {
 			return false, nil
 		}
 		utils.Poll(t, isServing, 10, time.Minute*1)
+
+		// Check cloud run jobs status
```
Comment (telpirion) [inline infra/test/integration/simple_example/simple_example_test.go:62]: issue: capitalize product names: "Cloud Run"

File: infra/test/integration/simple_example/simple_example_test.go [hunk 3]
```diff
@@ -58,6 +58,39 @@ func TestSimpleExample(t *testing.T) {
...
+		isJobCompletion := func() (bool, error) {
+			cloudRunJobs := gcloud.Run(t, ("beta run jobs list --format=json"), gcloudArgs).Array()
+			var jobResults []bool
```
Comment (telpirion) [inline infra/test/integration/simple_example/simple_example_test.go:65]: question: wouldn't we want to store job results by name? If we have 2 out of 5 jobs failing, we'll want to know which jobs failed and why.

**Review by telpirion — CHANGES_REQUESTED** [review]

File: infra/test/integration/simple_example/simple_example_test.go (hunk 1 above)
Comment (CienetStingLin) [reply infra/test/integration/simple_example/simple_example_test.go]: fixed in [12c25e4](https://github.com/GoogleCloudPlatform/terraform-large-data-sharing-golang-webapp/pull/19/commits/12c25e45d7043e134d12b0d5b91dd4002c41d660)

File: infra/test/integration/simple_example/simple_example_test.go (hunk 2 above)
Comment (CienetStingLin) [reply infra/test/integration/simple_example/simple_example_test.go]: fixed in [12c25e4](https://github.com/GoogleCloudPlatform/terraform-large-data-sharing-golang-webapp/pull/19/commits/12c25e45d7043e134d12b0d5b91dd4002c41d660)

File: infra/test/integration/simple_example/simple_example_test.go (hunk 3 above)
Comment (CienetStingLin) [reply infra/test/integration/simple_example/simple_example_test.go:65]: In this case, the test serves as an indicator which helps to ensure that the job is not in an active state before proceeding with CFT teardown.

**Review by telpirion — APPROVED** [review]

**Pull request merged**